### DISCUSSION TITLE
Store DateTime with Milliseconds (in PostgreSQL)?

### DISCUSSION THREAD

**Discussion by Alwinator** [mikro-orm/mikro-orm#3128] [Q&A] [answered]
By default a DateTimeType is translated to a `timestamp(0) with time zone` in PostgreSQL. Is there something like a DateTimeWithMillisecondsType that also stored milliseconds (`timestamp(3) with time zone`)

**Comment by Alwinator** [comment, marked as answer]
```ts
@Property({ type: DateTimeType, length: 3 })
```

**Reply by B4nan** [reply]
You don't even need to specify the type, it is an alias for `type: 'Date'` which is what you get with reflect metadata just by specifying the property type:

```ts
@Property({ length: 3 })
created: Date = new Date(); // its important to state the type explicitly for reflect metadata to work
```

**Reply by Alwinator** [reply]
Oh, that is great, thanks!
